source/funkin/play/notes/notekind/NoteKindManager.hx

```diff
@@ -10,7 +10,7 @@ import funkin.play.notes.notekind.NoteKind.NoteKindParam;
 
 class NoteKindManager
 {
-  static var noteKinds:Map<String, NoteKind> = [];
+  public static var noteKinds:Map<String, NoteKind> = [];
```

Comment:
Why is this public? If you are trying to access it from another class then use `@:privateAccess`.